openhtf/__init__.py

```diff
@@ -303,8 +303,8 @@ class PhaseOptions(mutablerecords.Record('PhaseOptions', [], {
   """Options used to override default test phase behaviors.
 
   Attributes:
-    name: Override for the name of the phase. Can be formatted similar to
-        measurements using with_args() to distinguish multiple similar phases.
+    name: Override for the name of the phase. Can be formatted in many different
+        formats defined in util.format_string.
```

Comment:
wording nit: s/formatted in many different formats/formatted several different ways/ (or the like).